demo/package.json

```diff
@@ -9,13 +9,13 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "bluebird": "2.9.34",
-    "body-parser": "1.13.3",
-    "connect": "3.4.0",
-    "connect-chain": "0.0.1",
-    "lodash": "3.10.1",
-    "method-override": "2.3.5",
-    "serve-static": "1.10.2",
-    "swagger-ui-dist": "3.6.1"
+    "bluebird": "*",
```

Comment:
Seems fine, could pin major versions instead to avoid accidentally upgrading to an incompatible version in the future.

Reply:
Updated, locked in version numbers.